classes/Search.php

```diff
@@ -349,7 +349,7 @@ public static function find(
         $sqlScore = '';
         if (!empty($scoreArray) && is_array($scoreArray)) {
             $sqlScore = ',( ' .
-                'SELECT SUM(weight) ' .
+                'SELECT 0 - SUM(weight) ' .
```

Comment:
could you please explain this fix ? here you'll get a negative value instead of a positive one, right ? if yes, why is that ?

Reply:
You are correct the idea is to switch the order so that the most relevant matches have a lower number not a higher one. This is needed because the "Relevance" sort by option on the search page uses ascending ordering.

The search score uses the alias `position` and other product lists assume that lower numbers come first. By modifying the behaviour to be the same as other product lists the can problem can be fixed with minimal changes. It also works on the 8.1 and dev branches.

Hope that helps.

Reply:
thank you for the explanation 👍